sdk/tests/deqp/functional/gles3/es3fNegativeBufferApiTests.js

```diff
@@ -481,12 +481,13 @@ goog.scope(function() {
         testGroup.addChild(new es3fApiCase.ApiCaseCallback(
             'copy_buffer_sub_data', 'Invalid gl.copyBufferSubData() usage', gl,
             function() {
-                var data = new Float32Array(32 * 32);
+                var data = new Float32Array(32);
```

Comment:
By reading the original deqp test (original test is correct, except it allocate more data than necessary), I think the bug here is we only need to supply 32 bytes to the bufferData call.

So the simplest fix is to change the array size to 32/Float32Array.BYTES_PER_ELEMENT.